modules/images-describing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-describing-image-registry-settings-hcp_{context}"]
+= Describing image registry settings for ROSA with HCP
+
+
+You can block any registry, and optionally an individual repository within a registry, by editing the `image.config.openshift.io/cluster` custom resource (CR). {product-title} applies the changes to this CR to all nodes in the cluster.
+
+When pulling or pushing images, the container runtime searches the registries listed under the `registrySources` parameter in the `image.config.openshift.io/cluster` CR. If you created a list of registries under the `blockedRegistries` parameter, the container runtime does not search those registries. All other registries are allowed.
+
+[WARNING]
+====
+To prevent pod failure, do not add the `registry.redhat.io` and `quay.io` registries to the `blockedRegistries` list, as they are required by payload images within your environment.
+====
+
+.Procedure
+
+. Describe ROSA with HCP clusters with image registry:
++
+[source,terminal]
+----
+$ rosa describe cluster -c <cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+Name:                       magchen-hcp
```

Comment:
Whenever we do describe, we need to replace the defined parameters with example, like
![image](https://github.com/user-attachments/assets/0abd4cda-7267-4dfa-be22-13afe9c5718c)
![image](https://github.com/user-attachments/assets/5022559d-181f-4050-b064-09eb91ba0f7a)